src/Chainweb/RestAPI/Orphans.hs

```diff
@@ -334,6 +334,9 @@ instance ToSchema (TransactionProof SHA512t_256) where
 instance ToSchema (TransactionOutputProof SHA512t_256) where
     declareNamedSchema _ = return $ NamedSchema (Just "TransactionOutputProof") $ byteSchema
 
+instance ToSchema TransactionOutputProofB64 where
+    declareNamedSchema _ = return $ NamedSchema (Just "TransactionOutputProofB64") byteSchema
```

Comment:
Is this proof fixed-length? If so you can use the  `minLength` and `maxLength` lenses to specify that in the Swagger.

Reply:
Discussed, since this is cosmetic, we'll do this separately if at all